### DISCUSSION TITLE
Trackers input type unclear

### DISCUSSION THREAD

**Discussion by YiannisParask** [mikel-brostrom/yolo_tracking#917] [Q&A] [answered]
In the following block of code you ask for a **numpy** array of detections in the format [[x1,y1,x2,y2,score],[x1,y1,x2,y2,score],...] and later on you try to convert it to **numpy** again. Wouldn't that raise an error?

```
xyxys = dets[:, 0:4]
        scores = dets[:, 4]
        clss = dets[:, 5]
        
        classes = clss.numpy()
        xyxys = xyxys.numpy()
        scores = scores.numpy()
```

**Comment by mikel-brostrom** [comment, marked as answer]
What should be passed to all trackers is torch arrays. I need to find time to fix the docstrings.

**Reply by YiannisParask** [reply]
Thanks for answering! My knowledge of torch is very limited and i've been trying to figure out a solutions for days.

**Comment by mikel-brostrom** [comment]
Just an update. All inputs are now `numpy.ndarray`. Check out: https://github.com/mikel-brostrom/yolo_tracking#custom-object-detection-model-example

**Reply by YiannisParask** [reply]
Thanks for the update 😃
